.pronto.yml

```diff
@@ -0,0 +1,8 @@
+rubocop:
+  # Map of RuboCop severity level to Pronto severity level
+  severities:
+    refactor: info
+    warning: error
+
+  # Enable suggestions
+  suggestions: true
```

Comment:
Careful with not leaving an empty line at the end of files. You can configure your editor to do this for you automatically